### DISCUSSION TITLE
XTTS function requires filling in a link, how to generate a link? The XTTS project did not generate a link scheme. Can someone give me a code?

### DISCUSSION THREAD

**Discussion by windkwbs** [LostRuins/koboldcpp#921] [General]
XTTS function requires filling in a link, how to generate a link? The XTTS project did not generate a link scheme. Can someone give me a code?

**Comment by LostRuins** [comment]
If you are running it locally, the link will appear in the XTTS terminal window.
![image](https://github.com/LostRuins/koboldcpp/assets/39025047/8e1e4607-ff87-4ba7-a440-22542a4d9eec)

Usually, it is http://localhost:8020
